### ISSUE TITLE
Type Error IFormControl

### ISSUE THREAD

**Issue by Yuri-Lima** [jorroll/solid-forms#8]
Hi, not sure if that is correct. I'm just stating dev. with solidjs and im facing that issue:

Error Type:
![Error1](https://user-images.githubusercontent.com/9964218/221119692-fa3c02d4-8dfc-4d22-8b2a-b3c090fa47df.PNG)

the solution i found: 
![Error3](https://user-images.githubusercontent.com/9964218/221119814-aaf3bbdd-3122-4481-9a74-c497fe241857.PNG)

Let me know if it's right. Thanks

**Comment by jorroll** [comment]
Ah, are you referring to one of the examples in this article I wrote about SolidForms? https://dev.to/johncarroll/awesome-forms-with-solidjs-18gi

What you've done looks like the correct solution. The error you are running into is coming from typescript. It's indicating that the way you typed the controls was conflicting. If you're working in a project with the `strictNullChecks` typescript compiler option turned off, then the code in question wouldn't error (I think).

I wrote those examples using stackblitz which, I think, has that compiler option turned off by default. That's probably why I didn't run into it.
